pkg/providers/nutanix/kubectl.go

```diff
@@ -0,0 +1,33 @@
+package nutanix
+
+import (
+	"context"
+
+	etcdv1beta1 "github.com/mrajashree/etcdadm-controller/api/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	kubeadmv1beta1 "sigs.k8s.io/cluster-api/controlplane/kubeadm/api/v1beta1"
+
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/executables"
+	"github.com/aws/eks-anywhere/pkg/types"
+)
+
+type ProviderKubectlClient interface {
```

Comment:
This looks like a copy-n-paste interface with several of the behaviors not relevant to the Nutanix provide; lets clean it up!